specs/phase0/beacon-chain.md

```diff
@@ -1732,8 +1732,9 @@ def process_randao(state: BeaconState, body: BeaconBlockBody) -> None:
 ```python
 def process_eth1_data(state: BeaconState, body: BeaconBlockBody) -> None:
     state.eth1_data_votes.append(body.eth1_data)
-    if state.eth1_data_votes.count(body.eth1_data) * 2 > EPOCHS_PER_ETH1_VOTING_PERIOD * SLOTS_PER_EPOCH:
-        state.eth1_data = body.eth1_data
+    if body.eth1_data.block_hash != Bytes32(): 
```

Comment:
Regardless of the change to default volte, should we have a check in here that the deposit count can't go backwards? It can only happen if enough validators don't follow the honest validator spec but would definitely be a very odd case to allow.